src/uikit.cs

```diff
@@ -8095,15 +8366,27 @@ interface UIButton : UIAccessibilityContentSizeCategoryImageAdjusting
 		[Export ("imageView", ArgumentSemantic.Retain)]
 		UIImageView ImageView { get; }
 
+		[TV (15,0), Watch (8,0), iOS (15,0), MacCatalyst (15,0)]
+		[NullAllowed, Export ("subtitleLabel", ArgumentSemantic.Strong)]
+		UILabel SubtitleLabel { get; }
+
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Ignored when 'Configuration' is used, override 'LayoutSubviews', call base, and set position views on your own.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Ignored when 'Configuration' is used, override 'LayoutSubviews', call base, and set position views on your own.")]
```

Comment:
Same: not deprecated on the watch?

Reply:
This type is not available on Watch

```
#if !WATCH
	[BaseType (typeof (UIControl))]
	interface UIButton : UIAccessibilityContentSizeCategoryImageAdjusting
#if IOS
```

Reply:
There's am `[Watch (8, 0)]` on the previous member which should be removed then, at best it's confusing.